+++
# Experience widget.
widget = "experience"  # Do not modify this line!
active = true  # Activate this widget? true/false

title = "Research Experience"
subtitle = ""

# Order that this section will appear in.
weight = 8

# Date format for experience
#   Refer to https://sourcethemes.com/academic/docs/customization/#date-format
date_format = "January 2006"

# Experiences.
#   Add/remove as many `[[experience]]` blocks below as you like.
#   Required fields are `title`, `company`, and `date_start`.
#   Leave `date_end` empty if it's your current employer.
#   Begin/end multi-line descriptions with 3 quotes `"""`.

[[experience]]
  title = "PhD Student"
  company = "Ludwig Cancer & UC San Diego<br/>(advisor: Bing Ren)"
  company_url = "http://renlab.sdsc.edu/renlab_website/"
  location = "La Jolla, CA"
  date_start = "2015-09-30"
  date_end = ""
  description = """
  * Developing snapR, a bioinformatics package that facilitates pre-processing, clustering, annotation for single-nucleus ATAC-seq analysis (in progress).
  * Constructed a cellular taxonomy of motor cortex in adult mouse brain using single cell ATAC sequencing.
  * Applied Latent Dirichlet Allocation (LDA) to single cell ATAC-seq data to identify cell-type specific regulatory modules.
  * Developed network-based analysis methods for analyzing single cell Hi-C datasets.
  * Characterized the dynamic of genome structure at the level of single cell during early neuronal differentiation.   
  * Developed and Applied multiplexed single nucleus ATAC-seq to delineate cis-regulatory landscapes of constitute cell types in a frozen complex tissue ([Nature Neurosciences](https://www.nature.com/articles/s41593-018-0079-3)).
  * Invented PLAC-seq (aka HiChIP) to map long-range chromatin interaction, improving the cost-efficiency by at least 10 fold compared to the previous method ([Cell Research](https://www.nature.com/articles/cr2016137)).
  * Developed CREST-seq, a titling deletion based CRISPR screening method to identify DNA regulatory elements, revealing enhancer elements for OCT4 gene ([Nature Methods](https://www.nature.com/articles/nmeth.4264)).
  * Applied convolutional neural networks to predict enhancer target genes using multi-omics data.
  """
  
[[experience]]
  title = "Summer Intern"
  company = "Illumina (advisor: Felix Schlesinger)"
  company_url = "https://www.illumina.com/"
  location = "San Diego, CA"
  date_start = "2015-07-01"
  date_end = "2015-09-30"
  description = """  
  * Developed a RNA-seq aligner within 2 months to detect structural variants in breast cancer using targeted RNA-seq, resulting in Illumina BaseSpace application and integrated into TruSight RNA/Tumor Fusion products (https://github.com/r3fang/TaFuCo)
  """

[[experience]]
  title = "Rotation Student"
  company = "Salk Institute for Biological Studies (advisor: Joe Ecker)"
  company_url = "https://www.salk.edu/scientist/joseph-ecker/"
  location = "La Jolla, CA"
  date_start = "2014-09-30"
  date_end = "2014-12-30"
  description = """
  * Analyzed whole-genome bisulfite sequencing data.
  * Developed an algorithm to identify DNA methylation footprint.
  """

[[experience]]
  title = "Research Assistant"
  company = "Duke University<br/>(advisor: Reluca Gordan)"
  company_url = "https://genome.duke.edu/directory/cbb-faculty-gcb-faculty/raluca-gord%C3%A2n-phd"
  location = "Durham, NC"
  date_start = "2013-11-11"
  date_end = "2014-04-30"
  description = """
  * Applied machine learning techniques to predict transcription factor binding specificity using protein binding microarray data.
  """

[[experience]]
  title = "Research Assistant"
  company = "Chinese Academy of Sciences<br/>(advisor: Zhihua Zhang)"
  company_url = "http://english.cas.cn/"
  location = "Beijing, China"
  date_start = "2012-10-01"
  date_end = "2013-11-01"
  description = """
  * Developed a motif discovery pipeline that identified three CTCF binding motifs and revealed distinctive functional enrichments between the three motifs by integrative analysis of various "omics" datasets.
  * Discovered that methylation status at 12th bp of the CTCF core motif might determine its function ([BMC Genomics](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1824-6)).  
  """
+++
